tests/data/scripts_pythondistdeps/test-requires.yaml

```diff
@@ -0,0 +1,91 @@
+setuptools:
+    wheel:
+        41.6.0: [2.7, 3.7, 3.9]
```

Comment:
Is it possible to have some tests for 3.10 as well to test double digits?

Reply:
Actually it is, for "sdist" I'm making the version up as I like. Ok, why not.

Reply:
And I found a problem in the test-data getting script :laughing:

Reply:
Ugh, yaml takes `3.10` as a float and thus returns `3.1`. I guess I'll have to quote all the values.